sample/src/main/java/com/davidecirillo/multichoicesample/sampleToolbar/SampleToolbarActivity.java

```diff
@@ -21,9 +21,15 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.util.ArrayList;
+import java.util.List;
 
 import butterknife.BindView;
 import butterknife.OnClick;
+import rx.Observable;
+import rx.Subscriber;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.functions.Func1;
+import rx.schedulers.Schedulers;
 
```

Comment:
My fault i didn't mean to push the sample just the library, the old sample from your library did have rx and i used it again to try to simulate network connection since i couldn't reproduce the issue somehow. I was kind in a hurry sorry